src/app/components/client/AutoSignIn.tsx

```diff
@@ -9,7 +9,7 @@ import { ReactNode, useEffect } from "react";
 
 export const AutoSignIn = (): ReactNode => {
   useEffect(() => {
-    void signIn("fxa", { callbackUrl: "/user/breaches" });
+    void signIn("fxa");
```

Comment:
without the callbackUrl, does it just go back to the original url?
I know we have another sign in button component. Is the goal to keep that one with redirect?

Reply:
Yes, without `callbackUrl` it redirects to the original URL. This `AutoSignIn` component is triggering sign-in when visiting a route that needs authentication. The `SignInButton` is an actual button that we use on the landing page and should direct to the dashboard.